packages/core/src/Table/stories/TableSection/TableEditable.tsx

```diff
@@ -432,72 +471,228 @@ const Table = <T extends Data>({
           className={classes.tableRowEditable}
           {...row?.getRowProps()}
         >
-          {cols.map((col, idx) => {
-            const last = cols.length - 1 === idx;
-
-            return (
-              <HvTableCell
-                className={classes.tableCellEditable}
-                key={`editable_row_cell-${col.accessor as string}`}
-                colSpan={last ? 3 : 1}
-                {...row?.cells[idx].getCellProps()}
+          <HvTableCell
+            className={classes.tableCellEditable}
+            key={`editable_row_cell-${cols[0].accessor as string}`}
+            colSpan={1}
+            {...row?.cells[0].getCellProps()}
+          >
+            <div className={edit ? undefined : classes.slide}>
+              <div className={classes.tableCellContent}>
```

Comment:
We need to review this. The rendering of the cell should be responsibility of the `columns`'s `Cell` property, not hard-coded into the "generic" Table.

If we can't do that, at least this should be refactored into a separate component - it's a lot of code!